### PR TITLE
Bugfix/cs 40469 - Replaced http calls with axiosInstace of Sdk for marketplace apps in both import and export

### PR DESCRIPTION
**Pull request by cs-raj**
Version Bump: (this will go with import export rewrite)

### CODE REVIEW

**Review by antonyagustine — APPROVED** [review]

File: packages/contentstack-export/src/export/modules-js/marketplace-apps.js [hunk 1]
```diff
@@ -44,7 +44,10 @@ module.exports = class ExportMarketplaceApps {
     }
 
     this.developerHubBaseUrl = this.config.developerHubBaseUrl || (await getDeveloperHubUrl(this.config));
-
+    this.appSdkAxiosInstance = await managementSDKClient({
+      host: this.developerHubBaseUrl.split('://').pop(),
+      endpoint: this.developerHubBaseUrl,
```
Comment (antonyagustine) [inline packages/contentstack-export/src/export/modules-js/marketplace-apps.js:49]: @cs-raj Can you confirm for me, should we need to pass both the host and endpoint.? Either one of them is enough right.?

File: packages/contentstack-export/src/utils/marketplace-app-helper.ts [hunk 2]
```diff
@@ -56,8 +55,16 @@ export const getStackSpecificApps = async (params: {
   skip: number;
 }) => {
   const { developerHubBaseUrl, httpClient, config, skip } = params;
-  return httpClient
-    .get(`${developerHubBaseUrl}/installations?target_uids=${config.source_stack}&skip=${skip}`)
+  const appSdkAxiosInstance = await managementSDKClient({
+    host: developerHubBaseUrl.split('://').pop(),
+    endpoint: developerHubBaseUrl,
```
Comment (antonyagustine) [inline packages/contentstack-export/src/utils/marketplace-app-helper.ts:60]: @cs-raj Chek this ass well

File: packages/contentstack-import/src/import/modules-js/marketplace-apps.js [hunk 3]
```diff
@@ -61,7 +61,10 @@ module.exports = class ImportMarketplaceApps {
 
     this.developerHubBaseUrl = this.config.developerHubBaseUrl || (await getDeveloperHubUrl(this.config));
     this.client = await managementSDKClient({ endpoint: this.developerHubBaseUrl });
-
+    this.appSdkAxiosInstance = await managementSDKClient({
+      host: this.developerHubBaseUrl.split('://').pop(),
+      endpoint: this.developerHubBaseUrl,
```
Comment (antonyagustine) [inline packages/contentstack-import/src/import/modules-js/marketplace-apps.js:66]: @cs-raj Chek this ass well

File: packages/contentstack-import/src/utils/marketplace-app-helper.ts [hunk 4]
```diff
@@ -16,10 +16,18 @@ export const getAllStackSpecificApps = async (
   httpClient: HttpClient,
   config: ImportConfig,
 ) => {
-  return await httpClient
-    .get(`${developerHubBaseUrl}/installations?target_uids=${config.target_stack}`)
-    .then(({ data }) => data.data)
-    .catch((error) => log(config, `Failed to export marketplace-apps ${formatError(error)}`, 'error'));
+  const appSdkAxiosInstance = await managementSDKClient({
+    host: developerHubBaseUrl.split('://').pop(),
+    endpoint: developerHubBaseUrl,
```
Comment (antonyagustine) [inline packages/contentstack-import/src/utils/marketplace-app-helper.ts:21]: @cs-raj Chek this ass well

File: packages/contentstack-export/src/export/modules-js/marketplace-apps.js (hunk 1 above)
Comment (cs-raj) [reply packages/contentstack-export/src/export/modules-js/marketplace-apps.js]: Done @antonyagustine used the host only

File: packages/contentstack-export/src/utils/marketplace-app-helper.ts (hunk 2 above)
Comment (cs-raj) [reply packages/contentstack-export/src/utils/marketplace-app-helper.ts]: Done @antonyagustine used the host only

File: packages/contentstack-import/src/utils/marketplace-app-helper.ts (hunk 4 above)
Comment (cs-raj) [reply packages/contentstack-import/src/utils/marketplace-app-helper.ts]: Done @antonyagustine used the host only

File: packages/contentstack-import/src/import/modules-js/marketplace-apps.js (hunk 3 above)
Comment (cs-raj) [reply packages/contentstack-import/src/import/modules-js/marketplace-apps.js]: Done @antonyagustine used the host only

**Review by shafeeqd959 — APPROVED** [review]

**Review by antonyagustine — APPROVED** [review]

**Pull request merged**